clients/rospy/src/rospy/core.py

```diff
@@ -143,14 +144,15 @@ def rospyerr(msg, *args):
 def rospywarn(msg, *args):
     """Internal rospy client library warn logging"""
     _rospy_logger.warn(msg, *args)
-    
+
 
 def _base_logger(msg, *args, **kwargs):
 
     name = kwargs.pop('logger_name', None)
     throttle = kwargs.pop('logger_throttle', None)
     level = kwargs.pop('logger_level', None)
     once = kwargs.pop('logger_once', False)
+    unique = kwargs.pop('unique', False)
```

Comment:
use the same name convention as the other kwargs - logger_unique.

IMO logger_throttle_identical is more self explanatory, but I'm indifferent.

Reply:
i'm less a fan logger_throttle_identical because it would imply logger_throttle should be renamed logger_throttle_time or logger_throttle_period

Reply:
came round to your way of thinking, renamed everything identical